<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 0, 57);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 100, 176);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_user_admin">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(4, 255, 0);color:rgb(117, 2, 2);font-size:12pt;</string>
        </property>
        <property name="text">
         <string>admin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
        </property>
        <property name="text">
         <string>New User Creation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_of_entrances" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 100, 176);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="customer_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="newUser_rights">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="marker2_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Rights</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="marker1_new_order_operator">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size: 10pt;</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="newUser_password">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="newUser_name">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color:rgb(224, 255, 188); font-size:13pt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createBtn">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 117, 119); color: rgb(126, 255, 141); font-size:13pt;</string>
     </property>
     <property name="text">
      <string>Create new user</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="feedback_label">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(11, 0, 57);color:rgb(255, 0, 0);font-size:12pt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="communications_entryfield_new_order_operator">
     <property name="styleSheet">
      <string notr="true">color: rgb(162, 0, 0); background-color: rgb(234, 234, 234); font: 700 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#1e1f22;&quot;&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#94558d;&quot;&gt;self&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#a9b7c6;&quot;&gt;.rights_list = [&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;login_viewing_rights&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#cc7832;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;Operator&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#cc7832;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;User&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#cc7832;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;Sales&amp;quot;, &amp;quot;developer&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#cc7832;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;controller&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#cc7832;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;ceo&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#cc7832;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#6a8759;&quot;&gt;&amp;quot;admin&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; color:#a9b7c6;&quot;&gt;]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
